pkgs/tools/typesetting/tex/texlive-new/combine.nix

```diff
@@ -158,7 +158,8 @@ in buildEnv {
     texlinks.sh "$out/bin" && wrapBin
     perl `type -P fmtutil.pl` --sys --refresh | grep '^fmtutil' # too verbose
     #texlinks.sh "$out/bin" && wrapBin # do we need to regenerate format links?
-    perl `type -P updmap.pl` --sys --syncwithtrees --force
+    echo "y" | perl `type -P updmap.pl` --sys --syncwithtrees --force
+    perl `type -P updmap.pl` --sys --force
```

Comment:
This second invocation fails for me due to missing `--syncwithtrees`. BTW, any hints how you found what was needed for you? When creating texlive-new, I could find no documentation describing what needs to be run and how, so I had to hack it together.